### ISSUE TITLE
Possibility to assign/remove an external IP addresses to a server

### ISSUE THREAD

**Issue by Sebi94nbg** [profitbricks/profitbricks-cli#8]
I can't find the option to assign an external IP address to a specific server. Is this option already available or is it really missing?

If it's awesome, it would be very awesome, if you could add it to the command to reserve new IP blocks. For example like this:

        $ profitbricks ipblock create --location [location] --size [size] --serverid [serverid]

Note: It also should be possible to remove such an IP from a server.

**Comment by jasminSPC** [comment]
IP addresses are assigned through the Network Interfaces See here: https://github.com/profitbricks/profitbricks-cli#create-nic

**Comment by Sebi94nbg** [comment]
Well, the parameter "--ip" of this command is used to set a local LAN IP address for the NIC, which will be displayed in the webinterface.

But I want to assign an external IP address to this NIC, which is NOT possible, because I can't assign it to the NIC. The assignment of the create nic command is shit, because I also could set "127.0.0.1" or "8.8.8.8" and it would work, because it's just a information field, which has no function...

**Comment by Sebi94nbg** [comment]
Any news regarding this issue?

**Comment by edevenport** [comment]
@Sebi94nbg - I'm going to allocate some time today to test assigning a public IP to a NIC. I'll provide more details when I have them.

**Comment by Sebi94nbg** [comment]
Great, thanks in advance!

**Comment by edevenport** [comment]
@Sebi94nbg - Now that #13 has been fixed, I was able to successfully set a public IP on the NIC using the CLI. The LAN has to be public and DHCP enabled on the NIC. I then allocated a public IP block and selected an IP from that block. Below is the command used to set the IP on the NIC.

    profitbricks nic update --datacenterid [dc_id] --serverid [server_id] --id [nic_id] --ip [ip_from_block]

It can take a few moments for the server to retrieve the new public IP address over DHCP. I was able to use the above command to switch between several different public IP addresses from my assigned block.
